src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+
+/**
+ * Storage class is used for storing data and loading data
+ */
+public class Storage {
+    private final String filePath;
```

Comment:
filePath is a constant.
```suggestion
    private final String FILE_PATH;
```

Reply:
Thank you for pointing out this!